huawei_lte_api/api/Net.py

```diff
@@ -23,6 +23,13 @@ def network(self) -> GetResponseType:
     def register(self) -> GetResponseType:
         return self._connection.get('net/register')
 
+    def set_register(self, mode: str, plmn: str, rat: str) -> SetResponseType:
+        return self._connection.post_set('net/register', OrderedDict((
```

Comment:
Could you please add usage info from your pull description here as method comment? like this: 
https://github.com/Salamek/huawei-lte-api/blob/2ff0b18b41f40232325a934ffc216eac26238ee5/huawei_lte_api/api/FileManager.py#L8-L14

thx

Reply:
Done.
I don't know if I did it correctly, I tried to document the best I could.